resolvers.ts

```diff
@@ -825,9 +826,9 @@ export const resolvers: Resolvers = {
           await fetchFromNextGen({
             customQuery: convertSequencingReadsQuery(context.params.query),
             customVariables: {
-              where: input.where,
-              orderBy: input.orderBy != null ? [input.orderBy] : [], // TODO: Migrate to array orderBy.
-              limitOffset: input.limitOffset,
+              where: input?.where,
```

Comment:
I'd prefer the function fail if `input` is nullish than send the wrong query to NextGen

Reply:
Actually I'm not even sure how this is compiling without the `?`...ok in that case I'm fine with it lol

Reply:
And I'm using `?` in other places already so I don't know what I'm talking about lol

Reply:
Since you pulled it's checking `== null` above now so this isn't needed